packages/babel-core/package.json

```diff
@@ -74,7 +74,16 @@
     "@types/gensync": "^1.0.0",
     "@types/resolve": "^1.3.2",
     "@types/semver": "^5.4.0",
-    "rimraf": "^3.0.0"
+    "rimraf": "^3.0.0",
+    "ts-node": "^10.9.1"
+  },
+  "peerDependencies": {
+    "@babel/preset-typescript": "^7.0.0"
+  },
+  "peerDependenciesMeta": {
+    "@babel/preset-typescript": {
+      "optional": true
+    }
```

Comment:
This is exactly how we should do it, but it's also a potential breaking change :slightly_frowning_face:  `peerDependenciesMeta` is only supported in npm 7+, and Node.js 14 comes with npm 6 preinstalled. There are also other places where we would need `peerDependenciesMeta`, but they are waiting for Babel 8.

Maybe we could `try/catch` requiring the preset without listing it as a peer dependency, even if this means that for now it won't work with PnP? Or we could only support loading through `ts-node` until Babel 8.

Reply:
I am a little hesitant about this, because even if the npm used by the user does not support `peerDependenciesMeta`, it should only install an additional plugin, and there will be no other side effects except for a very small hard disk occupation. 😕

Reply:
No, in old npm versions that don't support `peerDependenciesMeta` `@babel/preset-typescript` will not be installed, and instead it will be handled like a required peer dependency.

Reply:
I get it, this causes a warning to show up in some versions of npm, which is really less than ideal, thanks!